### PR TITLE
weekly blog post

### PR DESCRIPTION
**Pull request by adrianiurca**

### CODE REVIEW

File: _posts/2021-01-22-status-update.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+Please swing by our [office hours on Monday](https://puppet.com/community/office-hours/) on Slack if you want to directly contact us about anything module related.
+
+## Cloud CI
+
+Great news this week as we have tagged the v1.0.0 of our provisioning service that is used by the Cloud CI.
+This week there have been a couple of upstream issues with GCP and Github but overall, we had a bit over 3300 successful runs and around 150 failures on the nightly runs of the supported modules.
+Currently all our Github Actions acceptance tests are running with Puppet 5, Puppet 6 and Puppet 7 nightly. As we have been very happy with the stability and resiliance of the nightly Puppet 7 agent we will switch the Puppet 6 runs to use the nightly builds also. We have a [PR](https://github.com/puppetlabs/puppet_litmus/pull/373) prepared for that and with the next release of Puppet Litmus all our Github Action workflow matrices will change to use the Puppet 6 nightly builds.
+We are very thankful for all the help and support [Shelle](https://github.com/pup-shellee) offered us in making the service a secure and stable tool for spinning up the ephemeral testing machines.
```
Comment (DavidS) [inline _posts/2021-01-22-status-update.md:51]:
```suggestion
We are very thankful for all the help and support [Shellee](https://github.com/pup-shellee) offered us in making the service a secure and stable tool for spinning up the ephemeral testing machines.
```

File: _posts/2021-01-22-status-update.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+Hello! We have exciting news to share with you.
+Here is this week's update from the IAC Team:
+
+## Reaching IAC Team
+
+Here is an interesting blog post about how you can reach IAC team [Puppet IAC Team](https://puppetlabs.github.io/iac/team/2021/01/20/reaching-out.html)
+
+## Community Support Day
+
+The IAC Team processed 19 community PRs processed(12 toolings, 7 modules) also we have closed 32 module issues.
```
Comment (sanfrancrisko) [inline _posts/2021-01-22-status-update.md:25]:
```suggestion
The IAC Team processed 19 community PRs processed (12 tooling, 7 modules) also we have closed 32 module issues.
```

File: _posts/2021-01-22-status-update.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+We are very thankful for all the help and support [Shelle](https://github.com/pup-shellee) offered us in making the service a secure and stable tool for spinning up the ephemeral testing machines.
+We are touching upon three different layers (facade, backend and provision resources) to create appropriate firewalls which will helps us to tighten our security on Cloud CI provision service.
+
+## Introduction to PDKSync
+
+[Sheena][sheena] has written a great blog post this week giving an [overview of PDKSync]({% link _docs/pdksync.md %}), an invaluable tool that we use a lot on the team!
+
+## litmus release 0.22.0
+
+David Swan released his workaround to a long-standing bug in bolt which was only triggered by the concat test suite.
```
Comment (sanfrancrisko) [inline _posts/2021-01-22-status-update.md:60]:
```suggestion
[David Swan][davidswan] released his workaround to a long-standing bug in bolt which was only triggered by the concat test suite.
```

**Review by sanfrancrisko — COMMENTED** [review]
Just some minor comments, but generally LGTM 👍

**Review by sanfrancrisko — APPROVED** [review]

**Pull request merged**